<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>120</y>
          <width>141</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string>密码登录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>120</y>
          <width>141</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string>指纹登录</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>50</y>
          <width>241</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>选择您的登录方式</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_3">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>280</y>
          <width>301</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>260</y>
          <width>161</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>260</y>
          <width>161</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>回退</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>100</y>
          <width>111</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>登录密码</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>110</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>111</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>160</y>
          <width>111</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>160</y>
          <width>111</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>160</y>
          <width>111</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>SmartDevice</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>110</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_15">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>260</y>
          <width>151</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>111</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>20</y>
          <width>101</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>query sensor</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_4">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>30</y>
          <width>291</width>
          <height>371</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_11">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>100</y>
          <width>101</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>device data</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_5">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_6">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>90</y>
          <width>141</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_7">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>160</y>
          <width>141</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>151</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>类型</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>151</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>编号</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_14">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>151</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>操作</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_16">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>260</y>
          <width>101</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>添加指纹</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_17">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>260</y>
          <width>111</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>录入位置</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_8">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>260</y>
          <width>91</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_18">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>180</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>sql data</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_19">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>340</y>
          <width>111</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>录入等级</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_9">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>340</y>
          <width>91</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_20">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>340</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>删除指纹</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
